wof/WaterML_1_1.py

```diff
@@ -402,7 +402,7 @@ def quote_xml(inStr):
         pos = mo.end()
     s3 = s1[pos:]
     s2 += quote_xml_aux(s3)
-    return s2
+    return s2.decode(ExternalEncoding)
```

Comment:
This is fine as long as we know that everything using `quote_xml` expectes a decoded text and note bytes.
In my experience it is usually better to enconde once, use bytes everywhere, and decode once when outputting text to the end user.